docs/install/manage-add-ons-packages.md

```diff
@@ -0,0 +1,511 @@
+---
+myst:
+  html_meta:
+    "description": "Manage add-ons, packages, and processes"
+    "property=og:description": "Manage add-ons, packages, and processes"
+    "property=og:title": "Manage add-ons, packages, and processes"
+    "keywords": "Plone 6, manage, backend, add-ons, packages, processes, cookiecutter, pm2, Zope"
+---
+
+
+(manage-add-ons-packages-and-processes-label)=
+
+# Manage add-ons, packages, and processes
+
+This chapter assumes you have previously {doc}`installed Plone from its packages <install-from-packages>`.
+That documented process is a streamlined version.
+In this section, we discuss details of the installation process, so that you can customize your Plone installation.
+It also covers routine management tasks that a developer might perform.
+
+
+(manage-installation-details-label)=
+
+## Installation details
+
+{term}`Cookiecutter` creates projects from project templates.
+The cookiecutter [`cookiecutter-plone-starter`](https://github.com/collective/cookiecutter-plone-starter/) creates a Plone project that you can install using {term}`Make`.
+It generates files for installing and configuring both the frontend and backend.
+For the backend, it uses [`cookiecutter-zope-instance`](https://github.com/plone/cookiecutter-zope-instance) to generate configuration files for a Zope WSGI instance.
+
+
+(manage-backend-installation-details-label)=
+
+## Backend installation details
+
+Inside your project, open `backend/Makefile`.
+The `make` target `instance/etc/zope.ini` performs several tasks.
+
+```makefile
+bin/pip:
+	@echo "$(GREEN)==> Setup Virtual Env$(RESET)"
+	python3 -m venv .
+	bin/pip install -U "pip" "wheel" "cookiecutter"
+
+instance/etc/zope.ini:	bin/pip
+	@echo "$(GREEN)==> Install Plone and create instance$(RESET)"
+	bin/pip install Plone -c https://dist.plone.org/release/$(PLONE_VERSION)/constraints.txt
+	bin/cookiecutter -f --no-input --config-file instance.yaml https://github.com/plone/cookiecutter-zope-instance
+	mkdir -p var/{filestorage,blobstorage,cache,log}
+```
+
+-   Creates a virtual environment if one does not exist, then upgrades Python package management tools.
+-   Installs Plone using a given `constraints.txt` file for a specific version using `pip`.
+-   Creates or updates the Zope configuration from its `instance.yaml` file using `cookiecutter-zope-instance`.
+-   Creates specified directories, if they do not exist.
+
+As you may surmise, you can configure your Zope instance.
+You have two options for configuration.
```

Comment:
Developer does not need to make a decision on one option. `cookecutter-plone-starter`already generated /instance/etc/ with configuration files for Zope. These configuration files can be customized now.

Reply:
So... would deleting everything between:

```
-   Creates specified directories, if they do not exist.
```

...and...

```
(manage-zope-configuration-with-cookiecutter-zope-instance-label)=
```

and maybe inserting this:

```
You can configure your Zope instance using `make` as described in the next section.
```

...be the way to go?

Reply:
Fixed in 315dea1c08d0b75a2b52e6b4fd485f2f7005c1bb